examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+to receive real-time killmail updates for specific EVE Online systems.
+
+Dependencies:
+    pip install websockets asyncio json
+"""
+
+import asyncio
+import json
+import logging
+import signal
+import sys
+from typing import List, Optional, Set
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)